src/commands/hug.js

```diff
@@ -1,8 +1,66 @@
+import axios from 'axios';
+import Discord from 'discord.js';
 import DiscordCommand from './command.js';
 
 export default class HugCommand extends DiscordCommand {
-    async execute() {
-        const hug = this.getEmojiByName('HugSpin');
-        this.message.channel.send(`${hug}`);
+
+  async execute() {
+    if (!this.member) {
```

Comment:
so as noted above, this isn't checking if Roo is enabled in the discord server. If he isn't enabled, he won't get the message event to begin with - that would be a huge security concern otherwise :)

`this.message.member` is to check if the author of the message is still a member of the server/guild.

Reply:
as noted above, we probably can scrap this check entirely